### PR TITLE
add telos styles

### PR DESCRIPTION
**Pull request by bondarik1995**

### CODE REVIEW

File: app/app.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ import AccountProvider from 'containers/AccountProvider';
 
 // Load the favicon and the .htaccess file
 /* eslint-disable import/no-unresolved, import/extensions */
-import '!file-loader?name=[name].[ext]!./images/favicon.ico';
+// import '!file-loader?name=[name].[ext]!./images/favicon.ico';
```
Comment (sergeyi23) [inline app/app.js:35]: Remove commented line

File: app/components/Base/index.js [hunk 2]
```diff
@@ -21,15 +27,23 @@ const Base = styled.div`
       ? `0 0 0 1px rgba(${BORDER_PRIMARY_RGB}, 0.4) !important`
       : `none`};
   border-top-left-radius: ${({ bordered, topRightRadius, withoutBR }) =>
-    (bordered || topRightRadius) && !withoutBR ? '5px' : 'none'};
+    (bordered || topRightRadius) && !withoutBR
```
Comment (sergeyi23) [inline app/components/Base/index.js:30]: Can we find a way to reuse that expression instead of duplication?

File: app/components/Base/index.js [hunk 3]
```diff
@@ -41,22 +55,54 @@ const Base = styled.div`
 
   ${x =>
     x.position === 'top'
-      ? `border-top-left-radius: 5px; border-top-right-radius: 5px; border-bottom: 1px solid ${BORDER_SECONDARY};`
+      ? `border-top-left-radius: ${_get(
```
Comment (sergeyi23) [inline app/components/Base/index.js:58]: Can  we reuse this expression instead of duplication?

File: app/components/Dropdown/MenuStyled.js [hunk 4]
```diff
@@ -1,10 +1,16 @@
...
 
+import { singleCommunityStyles } from 'utils/communityManagement';
+
+const styles = singleCommunityStyles();
+
 const MenuStyled = styled.div`
   margin: 10px 0;
   padding: 0;
-  border-radius: 5px;
+  border-radius: ${_get(styles, 'buttonsBorderRadius', '5px')};
```
Comment (sergeyi23) [inline app/components/Dropdown/MenuStyled.js:13]: The same expression that can be potentially reused

File: app/app.js (hunk 1 above)
Comment (bondarik1995) [reply app/app.js]: removed

File: app/components/Base/index.js (hunk 2 above)
Comment (bondarik1995) [reply app/components/Base/index.js]: added three constants for border-radius (x, m, l sizes)

**Pull request merged**